src/oauth2.rs

```diff
@@ -24,15 +27,19 @@ const OAUTH2_YANDEX: Oauth2 = Oauth2 {
     init_token: "https://oauth.yandex.com/token?grant_type=authorization_code&code=$CODE&client_id=$CLIENT_ID&client_secret=58b8c6e94cf44fbe952da8511955dacf",
     refresh_token: "https://oauth.yandex.com/token?grant_type=refresh_token&refresh_token=$REFRESH_TOKEN&client_id=$CLIENT_ID&client_secret=58b8c6e94cf44fbe952da8511955dacf",
     get_userinfo: None,
+    mx_pattern: None,
 };
 
+const OAUTH2_PROVIDERS: [Oauth2; 1] = [OAUTH2_GMAIL];
```

Comment:
Maybe include Yandex here too? It has no effect now because `mx_pattern` is `None` for them, but then why have this array.

Reply:
- It's cleaner to iterate over an array despite it having only one element. I preferred having a hashMap str→Oauth2 but [map literals][1] are not that clean.
- I haven't researched Yandex MX pattern yet (if there is one).
- I try to have the PR focused on G Suite accounts.
- I plan to research more providers and submit them if a future PR.
- Yandex domain is statically resolved anyway, it won't even hit the mx lookup code.

[1]: https://stackoverflow.com/a/27582993